src/main/java/pl/onsight/wypozyczalnia/service/impl/CartServiceImpl.java

```diff
@@ -44,4 +54,41 @@ public void addDiscountToCart(Cart cart, double discount) {
     public void removeProductsFromCart(Cart cart) {
         cart.setProducts(new LinkedList<>());
     }
+
```

Comment:
to musi bbyc boolean a nie void? i wydaje mi sie ze dzialaloby bez linijki  cart.setProducts(products);